spdx-tools-java fails to build on amd64

The package spdx-tools-java fails to build from source with the following error:

```
	at java.desktop/java.awt.Font.getLineMetrics(Font.java:2505)
	at java.desktop/java.awt.font.TextMeasurer.initAll(TextMeasurer.java:239)
	at java.desktop/java.awt.font.TextMeasurer.<init>(TextMeasurer.java:168)
	at java.desktop/java.awt.font.LineBreakMeasurer.<init>(LineBreakMeasurer.java:311)
	at java.desktop/java.awt.font.LineBreakMeasurer.<init>(LineBreakMeasurer.java:278)
	at org.spdx.spreadsheetstore.AbstractSheet.getNumWrappedLines(AbstractSheet.java:314)
	at org.spdx.spreadsheetstore.AbstractSheet.resizeRows(AbstractSheet.java:279)
	at org.spdx.spreadsheetstore.SpdxSpreadsheet.resizeRow(SpdxSpreadsheet.java:428)
	at org.spdx.spreadsheetstore.SpreadsheetStore.serialize(SpreadsheetStore.java:122)
	at org.spdx.tools.SpdxConverter.convert(SpdxConverter.java:209)
	at org.spdx.tools.SpdxConverter.convert(SpdxConverter.java:144)
	at org.spdx.tools.SpdxConverterTest.testJsonToXlsx(SpdxConverterTest.java:248)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:569)
	at junit.framework.TestCase.runTest(TestCase.java:177)
	at junit.framework.TestCase.runBare(TestCase.java:142)
	at junit.framework.TestResult$1.protect(TestResult.java:122)
	at junit.framework.TestResult.runProtected(TestResult.java:142)
	at junit.framework.TestResult.run(TestResult.java:125)
	at junit.framework.TestCase.run(TestCase.java:130)
	at junit.framework.TestSuite.runTest(TestSuite.java:241)
	at junit.framework.TestSuite.run(TestSuite.java:236)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:90)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:316)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:240)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:214)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
Caused by: java.lang.ExceptionInInitializerError: Exception java.lang.UnsatisfiedLinkError: /usr/lib/jvm/java-17-openjdk/lib/libfontmanager.so: libfreetype.so.6: cannot open shared object file: No such file or directory [in thread "main"]
	at java.base/jdk.internal.loader.NativeLibraries.load(Native Method)
	at java.base/jdk.internal.loader.NativeLibraries$NativeLibraryImpl.open(NativeLibraries.java:388)
	at java.base/jdk.internal.loader.NativeLibraries.loadLibrary(NativeLibraries.java:232)
	at java.base/jdk.internal.loader.NativeLibraries.loadLibrary(NativeLibraries.java:174)
	at java.base/jdk.internal.loader.NativeLibraries.findFromPaths(NativeLibraries.java:315)
	at java.base/jdk.internal.loader.NativeLibraries.loadLibrary(NativeLibraries.java:285)
	at java.base/java.lang.ClassLoader.loadLibrary(ClassLoader.java:2403)
	at java.base/java.lang.Runtime.loadLibrary0(Runtime.java:818)
	at java.base/java.lang.System.loadLibrary(System.java:2006)
	at java.desktop/sun.font.FontManagerNativeLibrary$1.run(FontManagerNativeLibrary.java:58)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:318)
	at java.desktop/sun.font.FontManagerNativeLibrary.<clinit>(FontManagerNativeLibrary.java:33)
	at java.desktop/sun.font.SunFontManager$1.run(SunFontManager.java:275)
	at java.desktop/sun.font.SunFontManager$1.run(SunFontManager.java:273)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:318)
	at java.desktop/sun.font.SunFontManager.initStatic(SunFontManager.java:273)
	at java.desktop/sun.font.SunFontManager.<clinit>(SunFontManager.java:268)
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Class.java:467)
	at java.desktop/sun.font.FontManagerFactory$1.run(FontManagerFactory.java:83)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:318)
	at java.desktop/sun.font.FontManagerFactory.getInstance(FontManagerFactory.java:75)
	at java.desktop/java.awt.Font.getFont2D(Font.java:526)
	at java.desktop/java.awt.Font.defaultLineMetrics(Font.java:2398)
	at java.desktop/java.awt.Font.getLineMetrics(Font.java:2505)
	at java.desktop/java.awt.font.TextMeasurer.initAll(TextMeasurer.java:239)
	at java.desktop/java.awt.font.TextMeasurer.<init>(TextMeasurer.java:168)
	at java.desktop/java.awt.font.LineBreakMeasurer.<init>(LineBreakMeasurer.java:311)
	at java.desktop/java.awt.font.LineBreakMeasurer.<init>(LineBreakMeasurer.java:278)
	at org.spdx.tools.compare.AbstractSheet.getNumWrappedLines(AbstractSheet.java:261)
	at org.spdx.tools.compare.AbstractSheet.resizeRows(AbstractSheet.java:226)
	at org.spdx.tools.compare.MultiDocumentSpreadsheet.importVerificationErrors(MultiDocumentSpreadsheet.java:426)
	at org.spdx.tools.CompareSpdxDocs.onlineFunction(CompareSpdxDocs.java:106)
	at org.spdx.tools.CompareSpdxDocsTest.testCompareDocumentsv22(CompareSpdxDocsTest.java:114)
	... 21 more

[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   CompareSpdxDocsTest.testCompareDocumentsv22:114 ? UnsatisfiedLink /usr/lib/jvm/java-17-openjdk/lib/libfontmanager.so: libfreetype.so.6: cannot open shared object file: No such file or directory
[ERROR]   CompareSpdxDocsTest.testCompareDocumentsv23:94 ? NoClassDefFound Could not initialize class sun.awt.X11FontManager
[ERROR]   CompareSpdxDocsTest.testDifferentDocuments:132 ? NoClassDefFound Could not initialize class sun.awt.X11FontManager
[ERROR]   SpdxConverterTest.testJsonToXls:224 ? NoClassDefFound Could not initialize class sun.awt.X11FontManager
[ERROR]   SpdxConverterTest.testJsonToXlsx:248 ? NoClassDefFound Could not initialize class sun.awt.X11FontManager
[INFO] 
[ERROR] Tests run: 24, Failures: 0, Errors: 5, Skipped: 0
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  52.696 s
[INFO] Finished at: 2025-02-04T07:33:50Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.2.5:test (default-test) on project tools-java: 
[ERROR] 
[ERROR] Please refer to /home/build/target/surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
failed to build package: unable to run package spdx-tools-java pipeline: unable to run pipeline: exit status 1
```

--- a/spdx-tools-java.yaml
+++ b/spdx-tools-java.yaml
@@ -1,7 +1,7 @@
 package:
   name: spdx-tools-java
   version: 1.1.8
-  epoch: 2
+  epoch: 3
   description: SPDX Command Line Tools using the Spdx-Java-Library
   copyright:
     - license: Apache-2.0
@@ -13,6 +13,8 @@ environment:
   contents:
     packages:
       - busybox
+      - fontconfig-dev
+      - freetype-dev
       - maven
       - openjdk-17-default-jdk
       - ttf-dejavu

